Jellyfin.Api/Models/UserDtos/ILyricsProvider.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.ObjectModel;
+using MediaBrowser.Controller.Entities;
+
+namespace Jellyfin.Api.Models.UserDtos
+{
+    /// <summary>
+    /// Interface ILyricsProvider.
+    /// </summary>
+    public interface ILyricsProvider
```

Comment:
I suggest this interface follows the existing `ISubtitleProvider` (Doesn't need `Search`)

put it in `MediaBrowser.Controller.Lyrics`